### ISSUE TITLE
Ansi "Â" character is decoded to &nbps. This is not respecting the word

### ISSUE THREAD

**Issue by MonicatheLearner** [microsoft/BotFramework-WebChat#4658]
Hi Team,
We have an issue while displaying the message coming  from bot. The word is not respected and breaks from anywhere. We are using styleOptions:**messageActivityWordBreak: 'break-word'** but then also not working.

We found that in QnMaker the Text has Ansi "Â" character,when we test this on chatbot , it is decoded to "&nbps", hence  breaks the word to fit the line without respecting the word.

Could you provide any suggestion to handle this on server side or client side?

Below find the screenshot for the issue, where I have highlighted the broken words with yellow(the words are collectively,and,members):
![image](https://user-images.githubusercontent.com/119510087/222382216-aa7935af-e49c-4bf9-bb09-1659d9e50f1d.png)

Also please find the HTML of the above message from bot where we find the &nbps.
![image](https://user-images.githubusercontent.com/119510087/222382800-2a7038c0-2f5d-481e-bd3b-781ffa8493d4.png)

**Comment by MonicatheLearner** [comment]
@stevkan Please find attached the Question/Answer set in excel format.
[qnaexport.xlsx](https://github.com/microsoft/BotFramework-WebChat/files/10882915/qnaexport.xlsx)

We are using  4.14.1 Webchat version. 
This is  non-React (WebChat.renderWebChat()), we are using javascript to integrate on our website.

This did not occur earlier.

**Comment by stevkan** [comment]
@MonicatheLearner - A colleague and I were able to repro the issue. (I no longer have a QnA resource but I was able to rely on his.) However, after digging deeper for a possible cause it seems that the issue is due to the text content that is being submitted/copied into QnA as answers. The source text appears to have hidden non-breaking spaces separating the words. Whatever method is being used for copying the text appears to be copying these over, as well. QnA doesn't perform any manipulation on the content submitted, it just passes it along. In this specific case, those spaces are being converted to `&nbsp` in the HTML.

We were able to test this by submitting a question against your QnA kb using Curl in Powershell. We then copied the seemingly normal looking text response into Notepad with word wrap enabled. The hidden non-breaking spaces produced one long string that carried beyond the edge of the app window. However, when we replaced the non-breaking space with a normal space between the first and second word, the string broke there. We then had the first word on the top line and the continuous string on the second line.

Next, we scrubbed the text of all the hidden non-breaking spaces, replaced the answer in QnA with the updated text, and tested again. The response in Web Chat then broke words in an expected manner.

So, the conclusion is this isn't an issue with Web Chat or QnA, but simply with the text you are providing. If you scrub the text of any unusual hidden characters, this should resolve your problem.
